src/main/drivers/pwm_output.c

```diff
@@ -31,12 +31,14 @@
 #define MULTISHOT_5US_PW    (MULTISHOT_TIMER_MHZ * 5)
 #define MULTISHOT_20US_MULT (MULTISHOT_TIMER_MHZ * 20 / 1000.0f)
 
-#define DSHOT_MAX_COMMAND 47
-
 static pwmWriteFuncPtr pwmWritePtr;
 static pwmOutputPort_t motors[MAX_SUPPORTED_MOTORS];
 static pwmCompleteWriteFuncPtr pwmCompleteWritePtr = NULL;
 
+#ifdef USE_DSHOT
+loadDmaBufferFuncPtr loadDmaBufferPtr;
```

Comment:
This may be overkill - simple if/switch will be faster and easier to read (at least until there more protocols appear)
But HAL/StdPeriph makes it about even ...